src/validate.js

```diff
@@ -95,6 +95,11 @@ function createErrorHandler(formData) {
       return {...acc, [key]: createErrorHandler(formData[key])};
     }, handler);
   }
+  if(Array.isArray(formData)){
+    return Object.assign([],formData.reduce((acc, item, index) => {
```

Comment:
I'm not sure `Object.assign` works with arrays; the `reduce` result should suffice in theory.